src/main.rs

```diff
@@ -171,7 +171,7 @@ fn run_spec(spec_text: &str, args: &[String], opts: Options) {
             print!(" ");
         }
 
-        let our_html = render_html(&test.input.replace("→", "\t"), opts);
+        let our_html = render_html(&test.input.replace("→", "\t").replace("\n", "\r\n"), opts);
```

Comment:
That line is part of the existing test harness. I removed it in commit 1682550 and then reinstated it in commit 24bb3da.